src/sections/Companies/style.module.scss

```diff
@@ -0,0 +1,19 @@
+.container {
+  background: linear-gradient(180deg, #9d7587 30%, #0a0842 160%);
+  min-height: 100vh;
+  padding: 40px;
+
+  .content {
+    position: relative;
+    z-index: 9999;
```

Comment:
seeking to learn: what is this z-index for? i'm slightly concerned about it affecting styles when we integrate the navbar component and it doesn't seem like we're dealing with too many layers here